src/main/resources/db/changelog/changelog-master.xml

```diff
@@ -248,4 +248,53 @@
                                  referencedColumnNames="id"
                                  referencedTableName="product"/>
     </changeSet>
+
+    <changeSet id="maybe_6_add_column_version" author="Yurii Vislobodskyi">
```

Comment:
if AbstractEntity will change version field from long -> short, tables should change bigint -> smallint